doc/iterations/2024.11/add_support_for_form_property_section_layout.adoc

```diff
@@ -0,0 +1,73 @@
+= Add support for form property section layout
+
+== Problem
+
+A property section in a form is made of a label, a help icon and a value area.
+Currently, the layout is always vertical: label and help icon are in div and the value area is below.
+
+== Key Result
+
+It will be possible as a specifier to choose if the layout is horizontal or vertical.
+
+== Solution
+
+The property section contains 
+
+* a "label" div which includes the label and the help icon.
+* a "value"div
+
+The content of the property section is layouted with CSS Grid.
+New attributes are added on form widget styles to specify the layout.
+It corresponds to CSS Grid options:
+
+ * gridTemplateColumns
+ * gridTemplateRows
+ * labelGridColumn : the grid options that applies to the label area.
+ It is a shortHand for `grid-column-start ` and `grid-column-end`.
+ * labelGridRow : the grid options that applies to the label area.
+ It is a shortHand for `grid-row-start ` and `grid-row-end`.
+ * valueGridColumn : the grid options that applies to the value area.
```

Comment:
I'm not sure that value is the best name widget would be better because it may not represent a value at all